.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+
+## Our Git Work Flow
+
+* for smaller projects we're essentially following the feature branch workflow that can be found [here](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+* For larger projects we're following the process discused below, which is better suited to ongoing projects with multiple releases and CI, you can find a detailed discussion [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)
+
+We use two primary branches.  `master` and `develop`.  `develop` is what you'll create pull requests against, and should be deployed to staging environments for testing. Once enough work has gone in to create a  deliverable & deployable piece of value, and all testing is complete we create a release branch (this is where we start adding version numbers, add documentation and do any prep related to the release), this then gets merged into master, which will then be deployed to a production environment,  we should merge `master` back into `development`.  Ideally `master` should always represent what is in production. 
+
+
+### Contributing to the project
+Always branch, and use the below naming schema for branches. Once you've completed the work for that branch, push it up to github and create a Pull Request against development.
```

Comment:
We should add something here about keeping feature branches as short-lived as possible (ideally a day or less). The longer a feature branch is open, the more possibility of merge hell.

This also helps to make tasks more granular so we can keeping shipping in small increments.